script.js

```diff
@@ -1,20 +1,66 @@
-function sum () 
-{var a = Number (document.getElementById ("numberinput1").value);
-var b = Number (document.getElementById ("numberinput2").value);
-var answersum = document.getElementById ("answersum").innerHTML = a + b;
-} 
-
-function subtract ()
-{var c = Number (document.getElementById ("numberinput3").value);
-var d = Number (document.getElementById ("numberinput4").value);
-var answersubtract = document.getElementById ("answersubtract").innerHTML = c - d;}
-
-function multiply ()
-{var e = Number (document.getElementById ("numberinput5").value);
-var f = Number (document.getElementById ("numberinput6").value);
-var answermultiply = document.getElementById ("answermultiply").innerHTML = e * f;}
-
-function divide ()
-{var g = Number (document.getElementById ("numberinput7").value);
-var h = Number (document.getElementById ("numberinput8").value);
-var answerdivide = document.getElementById ("answerdivide").innerHTML = g / h;}
\ No newline at end of file
+function calculate() {
+
+    let a = Number (document.getElementById ("value1").value);
+    let b = Number (document.getElementById ("value2").value);
+    let operator = document.getElementById ("operator").value;
+   
+    let result=0;
+   
+    if (operator == "add") {
+        result = a+b;
+    }
+    else if (operator == "subtract"){
+        result = a-b;
+    }
+    else if (operator == "multiply"){
+        result = a*b;
+    }
+    else if (operator == "divide"){
+        result = a/b;
+    }
+    document.getElementById ("result1").innerHTML = result;
+
+    switch (operator){
```

Comment:
am glad you put it in the switch statement, but ideally this should be in another function.

calculate1()  -> uses if statements

calculate2()  -> uses switch statement